scripts/local-save

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+#  with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+IMAGE_NAME="amazon/amazon-ecs-agent:latest"
+
+stage() {
+	local tarball="./out/ecs-agent.tar"
+	docker save -o "${tarball}" "${IMAGE_NAME}"
+}
+
+stage_windows() {
+	local ziproot="$(mktemp -d)"
+	local ziptgtdir="$(mktemp -d)"
```

Comment:
nit: I like "target" instead of "tgt"

Reply:
i had copied it from the other file -- will fix